authenticators/cf_authenticator.go

```diff
@@ -83,7 +84,16 @@ func (cfa *CFAuthenticator) Authenticate(metadata ssh.ConnMetadata, password []b
 		return nil, err
 	}
 
-	processGuid, err := cfa.checkAccess(logger, appGuid, index, string(cred))
+	parts := strings.Split(cred, " ")
+	tokenString := parts[1]
+	token, err := jwt.Parse(tokenString, func(token *jwt.Token) (interface{}, error) {
+		return []byte("Doesntmatter"), nil
```

Comment:
Could you just add a short comment saying why it doesnt matter? From looking at the jwt source code, it looks like I guess this uses HMAC  validation, which only requites a byte array and never uses the actual value. But it just might be nice to have a comment saying this is just required by this particular validation type or something. (You guys probably have more context on why this is here than we do)